Jenkinsfile

```
@@ -366,6 +373,29 @@ try {
     }
   }
 
+  stage('E2E tests') {
+    def parallelSteps = [:]
+    for (x in e2eFeatureFiles) {
+      def feature = x
+      parallelSteps[feature] = {
+        node {
+          checkoutCentreonBuild(buildBranch)
+          unstash 'tar-sources'
+          unstash 'cypress-node-modules'
+          def acceptanceStatus = sh(script: "./centreon-build/jobs/web/${serie}/mon-web-e2e-test.sh centos7 tests/e2e/cypress/integration/${feature}", returnStatus: true)
+          junit 'centreon-web*/tests/e2e/cypress/results/reports/junit-report.xml'
+          if ((currentBuild.result == 'UNSTABLE') || (acceptanceStatus != 0))
+            currentBuild.result = 'FAILURE'
+          archiveArtifacts allowEmptyArchive: true, artifacts: 'centreon-web*/tests/e2e/cypress/results/**/*.mp4|png'
```

does it works ? cause I don't see `or` operator in ant path matcher : https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/util/AntPathMatcher.html
you should test following syntax
```suggestion
          archiveArtifacts allowEmptyArchive: true, artifacts: 'centreon-web*/tests/e2e/cypress/results/**/*.mp4, centreon-web*/tests/e2e/cypress/results/**/*.png'
```